src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -14,26 +14,31 @@ import {
 import { getAttributesAccordionExpandedPanel } from './state/attributesAccordionSelector';
```

Comment:
in attributesAccordionSelector there is an import statement with a relative path going 4 levels up (`import Attributes, { AttributeWithOptions } from '../../../../types/Attributes';`). Please use a path starting from `src` instead.

Reply:
Done